src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -198,8 +209,22 @@ export function createFormConfig781a(iterations) {
         ),
         path: `disabilities/ptsd-secondary-incident-unit-assignment-${index}`,
         depends: isAnswering781aQuestions(index),
-        uiSchema: secondaryIncidentUnitAssignment.uiSchema(index),
-        schema: secondaryIncidentUnitAssignment.schema(index),
+        uiSchema: secondaryIncidentDescription.uiSchema(index),
+        schema: secondaryIncidentDescription.schema(index),
+      },
+      [`secondaryOtherSources${index}`]: {
+        title: `781a PTSD Other sources of information`,
```

Comment:
Should this fit the pattern with the other titles that use `setReviewTitle()`?

Also, it looks like you're using a string template instead of a string literal here. Interesting that the linter doesn't care. :thinking:

Reply:
Done!